cars/v1/vanilla/config.ini

```diff
@@ -7,4 +7,4 @@ docker_image=docker.elastic.co/elasticsearch/elasticsearch-oss
 # major version of the JDK that is used to build Elasticsearch
 build.jdk = 12
 # list of JDK major versions that are used to run Elasticsearch
-runtime.jdk = 12,11
+runtime.jdk = 12,11,13
```

Comment:
Given that [latest Elasticsearch](https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.4.2-linux-x86_64.tar.gz) versions already ship with Java 13 I think we can make this `13,12,11`.